protobuf/echo/Echo.hpp

```diff
@@ -207,6 +208,28 @@ namespace services
         infra::Function<void()> onGranted;
     };
 
+    template<class TServiceProxy>
+    class ServiceProxyResponseQueue : public TServiceProxy
+    {
+    public:
+        using Container = infra::BoundedDeque<infra::Function<void()>>;
+
+        template<std::size_t Max>
+        using WithStorage = infra::WithStorage<ServiceProxyResponseQueue, typename Container::template WithMaxSize<Max>>;
+
+        template<class... TArgs>
+        explicit ServiceProxyResponseQueue(Container& container, TArgs&&... args);
+
+        void RequestSend(infra::Function<void()> onRequestGranted) override;
+
+    private:
+        void ProcessSendQueue();
+
+        Container& container;
+
+        bool responseInProgress{ false };
```

Comment:
```suggestion
        void ProcessSendQueue();

    private:
        Container& container;
        bool responseInProgress{ false };
```

Reply:
Are you sure? I thought we were going to adhere to the sonarlint warnings to not have consecutive same-access specifier?

Reply:
Yes, I am sure. We are going to disable that Sonar warning. Our coding guidelines clearly describe that we use the access specifiers to distinguish different blocks of code and/or methods and variables.

Reply:
(y)